src/DateUtils.js

```diff
@@ -55,7 +55,7 @@ function twelveHourTime(date) {
     let hours = date.getHours() % 12;
     const minutes = pad(date.getMinutes());
     const ampm = date.getHours() >= 12 ? 'PM' : 'AM';
```

Comment:
whilst you're here, we should do i18n for 'PM'/'AM'. Apparently Korean is an example of a language that uses something else.